customize/configuration/custom_django_settings.py

```diff
@@ -0,0 +1,2 @@
+# This file is required to run the test project.
+# DO NOT COMMIT CHANGES TO THIS FILE.
```

Comment:
We should also be able to remove this and add it to the git ignore.

We should provide an empty `__init__.py` file as well.